azure-pipelines.yml

```diff
@@ -141,6 +141,16 @@ jobs:
         - Linux_x64
         - Windows_NT_x64
         - Windows_NT_x86
+      ${{ if eq(variables['Build.DefinitionName'], 'coreclr-outerloop-jitstress-isas-arm') }}:
```

Comment:
In this case then we just need to add and rename new build definitions.

Reply:
Do you mean add a new `coreclr-outerloop-jitstress-isas-arm` entry in AzDO and rename the `coreclr-outterloop-jitstressisas` job to `coreclr-outerloop-jitstress-isas-x86`?

Reply:
@hoyosjs, do we have a document describing how to create the new build-definitions for coreclr? (do we just create a new definition and point it to the azure-pipelines.yml; do we clone an existing job and just rename it; is there any additional steps required for coreclr in particular; etc...)

Reply:
Indeed, clone, set the names correctly, and in the triggers please disable the PR validation ones.

Reply:
Thanks. Done.

What cadence do we want these scheduled to run and how should they be staggered as compared to the other `outerloop` jobs? It looks like:
* `outerloop` is never scheduled, but is part of PR validation
* `outerloop-jitstress` is `22:00 Mon through Sun`
* `outerloop-jitstressregs` is `2:00 Sat and Sun`
* `outerloop-gcstress0x3-gcstress0xc` is `5:00 Sat and Sun`
* `outerloop-jitstress2-jitstressregs` is `9:00 Sat and Sun`
* `outerloop-gcstress-extra` is `13:00 Sat and Sun`
* `outerloop-r2r-extra` is `18:00 Sat and Sun`